README.md

```diff
@@ -1,4 +1,25 @@
 # rrfs-workflow
 
-Future home of the workflow for the Rapid Refresh Forecast System (RRFS)
+Workflow for the Rapid Refresh Forecast System (RRFS)
+
+
+## Build
+
+1. Clone the `dev-sci` branch of the authoritative repository:
+```
+git clone -b dev-sci https://github.com/NOAA-EMC/rrfs-workflow
+```
+
+2. Check out the external components:
+```
+cd rrfs-workflow
+./manage_externals/checkout_externals
+```
+
+3. Build the RRFS workflow:
+```
+./devbuild.sh -p=[machine]
+```
+where `[machine]` is `wcoss2`, `hera`, `orion`, or `jet`.
```

Comment:
As can be seen in this new README file, this PR makes the build steps work up to Step 3 on `wcoss2`, `hera` and `orion`.

Reply:
@chan-hoo Thank you so much for working on this so quickly. Since dev-sci will be used by both GSL and EMC, I add @hu5970 as a reviewer so that GSL can get an understanding of how the modification will affect the future update of RRFS-B on JET.

Reply:
Thanks. As described above, the rrfs-workflow does not work yet. This PR only makes the 'build' step work well. I've tested it on wcoss2, hera, and orion. I don't have access to Jet. Are there any other machines to be tested?

Reply:
@chan-hoo  I think the RRFS workflow for operations will only need to be supported on WCOSS2, Hera, Jet, and Orion.  So at some point we will want to remove the logic for the other non-supported platforms.